go/teams/teams.go

```diff
@@ -249,7 +249,7 @@ func (t *Team) Rotate(ctx context.Context) error {
 
 	// post the change to the server
 	payload := make(libkb.JSONPayload)
-	if err := t.postChangeItem(ctx, section, secretBoxes, libkb.LinkTypeRotateKey, nil, nil, payload); err != nil {
+	if err := t.postChangeItem(ctx, section, secretBoxes, nil, libkb.LinkTypeRotateKey, nil, nil, payload); err != nil {
```

Comment:
getting to be a lot of (nil) arguments to `postChangeItem`.  Maybe a struct would be clearer so it would be `postChangeItem(ctx, someArgStruct)`.  Thoughts?